modulefiles/jet.intel/fv3

```diff
@@ -20,7 +20,7 @@ module load impi/2018.4.274
 module load hdf5/1.10.4
 module load netcdf/4.6.1
 
-module use -a /lfs3/projects/hfv3gfs/GMTB/modulefiles/intel-18.0.5.274
+module use -a /lfs4/HFIP/hfv3gfs/software/modulefiles/intel-18.0.5.274/impi-2018.4.274
 
 ##
 ### NCEP libraries (download and build manually for the time being; https://github.com/NCAR/NCEPlibs)
```

Comment:
Since NCEPlibs team is rebuilding nceplibs, I expect we will change the nceplibs modules files later?

Reply:
Yes, as noted above in the description of the PR. For now, the SAR real-time parallels can proceed with this version (same as they used until now).